media/vp8_ivf.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: vp8_ivf
+  file-extension: ivf
+  endian: le
+  title: VP8 raw file
+  ks-version: 0.7
+  license: CC0-1.0
+doc-ref: https://wiki.multimedia.cx/index.php/IVF
+seq:
+## header start
```

Comment:
I usually prefer the code be self-documenting. Should you remove these comments and use subtypes instead?